system/services/database/adapters/MsSqlAdapter.cfc

```diff
@@ -0,0 +1,592 @@
+component singleton=true {
+
+// CONSTRUCTOR
+	public any function init() {
+		return this;
+	}
+
+// PUBLIC API METHODS
+	public string function escapeEntity( required string entityName ) {
+		var escaped = "";
+		var checkDot = FindNoCase(".", entityName, 1);
+		if(checkDot) {
+				var indexOfDot = checkDot;
+				var tempCol 	 = Mid(entityName, indexOfDot + 1, Len(entityName));
+				escaped = Replace(entityName, "." & tempCol, ".[" & tempCol & "]", "one");
+		}
+		else {
+				switch (arguments.entityName) {
```

Comment:
I'm not sure for the need/want of switch statement here. What is the purpose of this? I think that if `escapeEntity()` is called, the logic should _always_ escape it with`[]`, regardless of the name of the entity.